.github/workflows/functional_test.yaml

```diff
@@ -20,8 +20,8 @@ jobs:
       CI_INDEX_EVENTS: ci_events
       CI_INDEX_METRICS: ci_metrics
       CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
-      KUBERNETES_VERSION: v1.21.2
-      MINIKUBE_VERSION: v1.22.0
+      KUBERNETES_VERSION: v1.25.6
```

Comment:
This is a large increment for both and shouldn't be associated w/ a release commit imo. Looks like @jvoravong is trying the minikube version out in https://github.com/signalfx/splunk-otel-collector-chart/pull/678 so may be best to table until vetted?

Reply:
Okay, good to know. From the [release doc](https://docs.google.com/document/d/1oaTyz_hx-89Mtm5_IhkuMugESedeq5nJaJXr3XC9xVU/edit#bookmark=id.kun46krd7r1i) I was trying to update to a supported version, and was going to wait and see how our testing went. I noticed they were both really old versions, but wasn't sure why.

I've removed these updates.